docker-compose.yml

```
@@ -0,0 +1,27 @@
+version: '3.8'
+
+services:
+  backend:
+    build: ./backend
+    ports:
+      - "127.0.0.1:8000:8000"
+    env_file:
+      - .env
+    depends_on:
+      mongodb:
+        condition: service_started
+
+  mongodb:
```

В пайплайне проверяется наличие базы данных в docker compose по наличию сервиса с именем `db`. Лучше сразу так и назвать:
```suggestion
  db:
```